config/jobs/periodic/knative/operator/main/operator-main.gen.yaml

```diff
@@ -0,0 +1,66 @@
+periodics:
+  - name: knative-operator-main-periodic
+    labels:
+      preset-knative-powervs: "true"
+    decorate: true
+    cron: "20 0 * * *"
+    extra_refs:
+      - base_ref: main
+        org: valen-mascarenhas14
+        repo: knative-upstream-ci
+        workdir: true
+      - base_ref: main
+        org: knative
+        repo: operator
+        workdir: false
```

Comment:
The default value of `workdir` is false. Needn't mention.

Reply:
changed it